tests/test_e2e_70_kytos_stats.py

```diff
@@ -0,0 +1,237 @@
+
+class TestE2EKytosStats:
+    
+    def setup_method(self, method):
+        """Called at the beginning of each class method"""
+        self.net.start_controller(clean_config=True, enable_all=True)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER)
+        cls.net.start()
+        cls.net.restart_kytos_clean()
+        time.sleep(10)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def test_005_flow_stats(self):
+        """Test flow_stats"""
+ 
+        api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 1
+        assert '00:00:00:00:00:00:00:01' in data
+
+        api_url = KYTOS_STATS + '/flow/stats?dpid=00:00:00:00:00:00:00:01&dpid=00:00:00:00:00:00:00:02'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 2
+        assert '00:00:00:00:00:00:00:01' in data
+        assert '00:00:00:00:00:00:00:02' in data
+
+        api_url = KYTOS_STATS + '/flow/stats'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+
+        api_url = KYTOS_API + '/kytos/topology/v3/switches'
+        response = requests.get(api_url)
+        data_topp = response.json()
+        assert len(data) == len(data_topp['switches'])
+
+    def test_010_table_stats(self):
+        """Test table_stats""" 
+
+        api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 1
+        assert '00:00:00:00:00:00:00:01' in data
+
+        api_url = KYTOS_STATS + '/table/stats?dpid=00:00:00:00:00:00:00:01&dpid=00:00:00:00:00:00:00:02' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == 2
+        assert '00:00:00:00:00:00:00:01' in data
+        assert '00:00:00:00:00:00:00:02' in data
+
+        api_url = KYTOS_API + '/kytos/topology/v3/switches'
+        response = requests.get(api_url)
+        data_topp = response.json()
+        topo_switches = data_topp['switches']
+
+        api_url = KYTOS_STATS + '/table/stats?table=0'  
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert len(data) == len(topo_switches)
+        for sw in topo_switches:
+            assert len(data[sw]) == 1
+            assert '0' in data[sw]
+
+        api_url = KYTOS_STATS + '/table/stats?table=0&table=1'  
```

Comment:
Gretel, sorry if I missed something. At this point, you didn't add flows into Table 1. Thus, my question is: how would the kytos_stats napp return data for table 1 if we don't have flows for table 1?